src/App.tsx

```diff
@@ -15,48 +15,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
```

Comment:
Recommendation: capitalize enum keys
```suggestion
  None,
  ALPHABET,
  LENGTH,
```